### PR TITLE
Create docker.md

### PR DESCRIPTION
**Pull request by mariadb-pieterhumphrey**
creating docker.md page to elaborate our docker images.   must be updated every release.

### CODE REVIEW

**Comment by mariadb-pieterhumphrey** [comment]
@mariadb-mmontes for your review.  wasn't sure if exporter images were getting bumped in version for v0.0.30 so they are still v0.0.1

File: docs/docker.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        <tr>
+      <td>MariaDB MaxScale prometheus exporter</td>
+      <td><a href=https://hub.docker.com/repository/docker/mariadb/maxscale-prometheus-exporter-ubi/tags?page=&page_size=&ordering=&name=%20>mariadb/maxscale-prometheus-exporter-ubi</a></td>
+      <td><code>v0.0.1</code></td>
+      <td><code>amd64</code> <code>arm64</code></td>
+	  <td><code>docker pull docker-registry2.mariadb.com/mariadb/maxscale-prometheus-exporter-ubi:v0.0.1</code></td>
+    </tr>
+        <tr>
+      <td>Community Operator</td>
+      <td><a href=ghcr.io/mariadb-operator/mariadb-operator>ghcr.io/mariadb-operator/mariadb-operator</a></td>
```
Comment (mmontes11) [inline docs/docker.md:55]:
```suggestion
      <td><a href=https://github.com/mariadb-operator/mariadb-operator/pkgs/container/mariadb-operator>ghcr.io/mariadb-operator/mariadb-operator</a></td>
```
This links is broken

File: docs/docker.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Docker Images for MariaDB Operator
+
+The mariadb-operator uses a variety of docker images depending on how it’s configured and what mix of open source or commercial software you’d like to use. As only the latest version of MariaDB Community Server is supported, the community server version will increment frequently with only a best effort made to keep current with the latest release(s). Only MariaDB Enterprise Server offers support for older versions.
+
+> **Note**: Using Docker images other than the supported ones in this document is not recommended at this time.
```
Comment (mmontes11) [inline docs/docker.md:5]:
```suggestion
> [!NOTE]  
> Using Docker images other than the supported ones in this document is not recommended at this time.
```
Use GitHub enhanced markdown banner

**Review by mmontes11 — CHANGES_REQUESTED** [review]
Hey there @mariadb-pieterhumphrey !

Exporters are still on `v0.0.1`, so we are good.

Left a couple of suggestions/improvements, Other than that Let'sGetThisMerged! 👍🏻

File: docs/docker.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    <tr>
+      <td>MariaDB Community Server</td>
+      <td><a href=https://hub.docker.com/_/mariadb/tags?page=&page_size=&ordering=&name=11.4.3-ubi9>mariadb</a></td>
+      <td><code>11.4.3-ubi9</code></td>
+      <td><code>amd64</code> <code>arm64</code> <code>ppc64le</code> <code>s390x</code></td>
+	  <td><code>docker pull docker-registry1.mariadb.com/library/mariadb:11.4.3-ubi9</code></td>
+    </tr>
+    <tr>
+      <td>MariaDB Enterprise Server</td>
+      <td><a href=https://hub.docker.com/_/mariadb/tags?page=&page_size=&ordering=&name=11.4.3-ubi9>docker.mariadb.com</a><br>Login required, click link for instructions</td>
```
[the remark this answers is not in the document]
Comment (mariadb-pieterhumphrey) [reply docs/docker.md:27]: docker.mariadb.com redirects to the main docs page in question, so it does

File: docs/docker.md (hunk 1 above)
Comment (mariadb-pieterhumphrey) [reply docs/docker.md:55]: thank you nice catch

File: docs/docker.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mariadb-pieterhumphrey) [reply docs/docker.md:28]: oh, apologies I see what you mean.  yes, I should have had the ahref be docker.mariadb.com, not dockerhub

**Review by mmontes11 — APPROVED** [review]
LGTM! To save you the trouble of solving conflicts, I've added this docs to `release-v0.0.30` branch and added you as a co-author:
- https://github.com/mariadb-operator/mariadb-operator/blob/eafa45f81cd1777ddeff487bf4bb16ae54eec60e/docs/DOCKER.md

**Comment by mmontes11** [comment]
This has been added to `release-v0.0.30` branch:
- https://github.com/mariadb-operator/mariadb-operator/blob/release-v0.0.30/docs/DOCKER.md

Closing!

**Pull request closed without merging**